deploy/terraform/variables.tf

```diff
@@ -0,0 +1,36 @@
+variable "AWS_REGION" {
+  type = string
+  description = "AWS region to use for resources"
+  default = "us-east-1"
```

Comment:
It's OK to have this file, but values for all of these should be provided in the environment.